dist/index.d.ts

```diff
@@ -2039,6 +2035,10 @@ interface GPUAdapterInfo {
    * adapter.
    */
   readonly subgroupMaxSize: number;
+  /**
+   * Whether the adapter is a fallback adapter.
+   **/
+  readonly isFallbackAdapter: boolean;
```

Comment:
We should make this optional for now so that applications will handle the fact that it might not exist.
```suggestion
  readonly isFallbackAdapter?: boolean;
```

Reply:
Done